### ISSUE TITLE
View#write_image and Overlay

### ISSUE THREAD

**Issue by Fredosixx** [SketchUp/api-issue-tracker#904]
First, it is unclear whether view#wite_image should or not draw the overlay drawings. Currently it does...

Second, in any case it does not draw it correctly and seems to pave the image file with a tiling of the overlay drawings

![Scene_7942027](https://github.com/SketchUp/api-issue-tracker/assets/57754157/7328be4d-612d-4afc-945e-391c8dcbf986)

The minimum is to get clarification about this.

**Comment by DanRathbun** [comment]
I think this has been reported and perhaps already assigned a Jira issue.

My opinion is that it **can** be useful if it works correctly, but the [`View#write_image`]() method should get an additional Boolean `:overlays` options in the argument hash so coders can choose to include them, and so that current code is not affected. (Ie, the option should default to `false`, if not given so that extensions in the wild do not see them.)

It might be interesting if somehow coders could choose a specific overlay to include in the output whilst suppressing all others. I suppose this can be done now by iterating the overlays manager and switching those off that are not desired in the output.

**Comment by Fredosixx** [comment]
Actually the bug is the tiling. Disabling or enabling overlay works anyway, until we have it in the options.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DanRathbun** [comment]
Yes, but again it could be beneficial. Fredo gives an example in the 1st post. 

* Grabbing an image to post in a Bug Report.

Other uses:

* Images for an extension user guide
* Tutorial images
* Help Center pages

----

@Fredosixx What platform, resolution and display scaling are you seeing this on ?

**Comment by Fredosixx** [comment]
Windows 10. Resolution set at the size of the viewport or multiple. But Tiling seems to follow its own logic.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dphdave** [comment]
Another use case I am currently looking at is compiling a report of production information, where various overlays are used to show different properties of the components of a building, then images of the building with the various overlays enabled are used to assist fabrication and assembly.
